docs/docs/how-tos/multi-agent-multi-turn-convo-functional.ipynb

```diff
@@ -0,0 +1,430 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "a2b182eb-1e31-43c8-85b1-706508dfa370",
+   "metadata": {},
+   "source": [
+    "# How to add multi-turn conversation in a multi-agent application (functional API)\n",
+    "\n",
+    "!!! info \"Prerequisites\"\n",
+    "    This guide assumes familiarity with the following:\n",
+    "\n",
+    "    - [Multi-agent systems](../../concepts/multi_agent)\n",
+    "    - [Human-in-the-loop](../../concepts/human_in_the_loop)\n",
+    "    - [Functional API](../../concepts/functional_api)\n",
+    "    - [Command](../../concepts/low_level/#command)\n",
+    "    - [LangGraph Glossary](../../concepts/low_level/)\n",
+    "\n",
+    "\n",
+    "In this how-to guide, we’ll build an application that allows an end-user to engage in a *multi-turn conversation* with one or more agents. We'll create a node that uses an [`interrupt`](../../reference/types/#langgraph.types.interrupt) to collect user input and routes back to the **active** agent.\n",
+    "\n",
+    "The agents will be implemented as tasks in a workflow that executes agent steps and determines the next action:\n",
+    "\n",
+    "1. **Wait for user input** to continue the conversation, or\n",
+    "2. **Route to another agent** (or back to itself, such as in a loop) via a [**handoff**](../../concepts/multi_agent/#handoffs).\n",
+    "\n",
+    "```python\n",
```

Comment:
Why is this not in a cell?

Reply:
It's also missing an import for `task`

Reply:
will add an import, i am not sure we want the full executable code snippet though -- it's usually a shortened version that explains how to do X (and doesn't need to be a code cell for that same reason)